jbmc/src/java_bytecode/java_bytecode_language.h

```diff
@@ -56,6 +57,11 @@ Author: Daniel Kroening, kroening@kroening.com
   " --java-assume-inputs-non-null\n" \
   "                              never initialize reference-typed parameter to the\n" /* NOLINT(*) */ \
   "                              entry point with null\n" /* NOLINT(*) */ \
+  " --java-assume-inputs-interval [L:U] or [L:] or [:U]\n" \
+  "                              never initialize numerical primitive-typed parameter\n" /* NOLINT(*) */ \
+  "                              (byte, short, int, long, float, double) to an entry\n" /* NOLINT(*) */ \
```

Comment:
:bulb: I'd phrase this as the positive "only initialize primitive typed parameters to values within the specified range"

Reply:
Rephrased